scalopus_general/include/scalopus_general/map_tracker.h

```diff
@@ -107,10 +105,6 @@ class MapTracker
 private:
   std::unordered_map<Key, Value> mapping_;
   mutable std::shared_timed_mutex mutex_;  //! Mutex for the mapping container.
-
-protected:
-  //! Make constructor private such that we can ensure it is a singleton.
-  MapTracker() = default;
```

Comment:
This was pretty much a moot point, since subclassing from it negated these constructor removals.